resources/ec2_connect

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
Not a blocker for this PR, but will also need a powershell script for Windows clients.